tests/gui/components/svg-icon.spec.js

```diff
@@ -33,9 +34,9 @@ describe('Browser: SVGIcon', function() {
 
     it('should inline the svg contents in the element', function() {
       const icon = '../../../../../lib/gui/assets/etcher.svg';
-      let iconContents = fs.readFileSync(path.join(__dirname, '../../../lib/gui/assets/etcher.svg'), {
+      let iconContents = _.split(fs.readFileSync(path.join(__dirname, '../../../lib/gui/assets/etcher.svg'), {
         encoding: 'utf8'
-      }).split(/\r?\n/);
+      }), /\r?\n/);
```

Comment:
Since we're only replacing the first line, maybe a `replace()`, rather than a `split()` might be shorter & better to comprehend – and we could reuse `icon` and `iconContents`:

```js
let iconContents = fs.readFileSync( path.join( __dirname, icon ), 'utf8' )
iconContents = _.replace( iconContents, /^([^\n\r]+?)/, ( match, firstLine ) => {
  return `<!--${firstLine.slice(1)}-->`;
})
```